### PR TITLE
Make hasLabel a helper.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
The small helper `hasLabel` is implemented in several places. This PR make it a public helper.

### CODE REVIEW

File: prow/plugins/helpers.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+package plugins
+
+import (
+	"strings"
+
+	"k8s.io/test-infra/prow/github"
+)
+
+// HasLabel checks if label is in the label set "issueLabels".
+func HasLabel(label string, issueLabels []github.Label) bool {
```
Comment (cjwagner) [inline prow/plugins/helpers.go]: I think this belongs in Prow's `github` package, not the `plugins` package.

Comment (xiangpengzhao) [reply prow/plugins/helpers.go]: Done. Thanks!

**Comment by xiangpengzhao** [comment]
@cjwagner inline comment addressed. PTAL. Thanks!

**Comment by cjwagner** [comment]
/lgtm
Thanks for changing this!

**Pull request merged**